activemodel/test/cases/errors_test.rb

```diff
@@ -25,6 +25,7 @@ def self.human_attribute_name(attr, options = {})
     def self.lookup_ancestors
       [self]
     end
+
```

Comment:
This is diff noise, it would be nice if this was reverted to minimize the noise.